examples/cog-modeltransformation.js

```diff
@@ -12,11 +13,26 @@ register(proj4);
 const cogSource = new GeoTIFF({
   sources: [
     {
-      url: 'https://umbra-open-data-catalog.s3.amazonaws.com/sar-data/tasks/Tanna%20Island,%20Vanuatu/9c76a918-9247-42bf-b9f6-3b4f672bc148/2023-02-12-21-33-56_UMBRA-04/2023-02-12-21-33-56_UMBRA-04_GEC.tif',
+      url:
+        'https://umbra-open-data-catalog.s3.amazonaws.com/sar-data/tasks/Tanna%20Island,%20Vanuatu/' +
+        '9c76a918-9247-42bf-b9f6-3b4f672bc148/2023-02-12-21-33-56_UMBRA-04/2023-02-12-21-33-56_UMBRA-04_GEC.tif',
     },
   ],
 });
 
+const invert = (band, power) => ['-', 1, ['^', ['band', band], power]];
+const showTilesCheckbox = document.getElementById('show-tiles');
+const debugLayer = new TileLayer({
+  source: new TileDebug({source: cogSource}),
+  style: {
+    color: ['array', invert(1, 4), invert(2, 4), 0, ['band', 4]],
+  },
+  visible: showTilesCheckbox.checked,
+});
```

Comment:
```suggestion
const showTilesCheckbox = document.getElementById('show-tiles');
const debugLayer = new TileLayer({
  source: new TileDebug({source: cogSource}),
  visible: showTilesCheckbox.checked,
});
```
Nice way to change the colors, but let's try to keep the example's focus on what's in the title and description.